flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/dsv2/eventtime/util/CountNewsClicksData.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.examples.dsv2.eventtime.util;
+
+import org.apache.flink.datastream.api.ExecutionEnvironment;
+import org.apache.flink.datastream.api.stream.NonKeyedPartitionStream;
+import org.apache.flink.streaming.examples.dsv2.eventtime.CountNewsClicks;
+import org.apache.flink.streaming.examples.dsv2.eventtime.CountNewsClicks.NewsEvent;
+import org.apache.flink.streaming.examples.dsv2.util.IOUtil;
+
+import java.net.URL;
+import java.util.Objects;
+
+/** Sample data for the {@link CountNewsClicks} example. */
+public class CountNewsClicksData {
```

Comment:
Should not introduce such a custom class. Instead, we can use file source with csv format.